charts/kafka/values.yaml

```diff
@@ -246,7 +246,7 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  # storageClass:
+  storageClass: "kafka-persistence"
```

Comment:
Update the header. Need to add the crown copyright and change the 'Copied from: ...' to 'Originally based on ...'